datafusion/src/physical_plan/expressions/mod.rs

```diff
@@ -60,6 +60,7 @@ pub mod helpers {
 
 pub use approx_distinct::ApproxDistinct;
 pub use array_agg::ArrayAgg;
+pub(crate) use average::is_avg_support_arg_type;
```

Comment:
I don't think we need to export this function (aka no need to declare it `pub crate` in this function). Likewise below

Reply:
But `is_avg_support_arg_type` will be used in other model, for example `coercion_rule` model.
https://github.com/apache/arrow-datafusion/blob/42b219215f1962bbf529c06392f5c7c978f70cc4/datafusion/src/physical_plan/coercion_rule/aggregate_rule.rs#L75